examples/mugen/test/retrieval/test_video_clip.py

```diff
@@ -12,18 +12,47 @@
 
 
 class TestTextEncoder:
-    @pytest.fixture
-    def start(self):
+    @pytest.fixture(autouse=True)
+    def set_seed(self):
         set_rng_seed(1234)
-        text_input = ["raw text sample for testing encoder"]
+
+    @pytest.fixture
+    def utils(self, set_seed):
+        input_ids = torch.Tensor(
+            [
+                [101, 6315, 3793, 7099, 2005, 5604, 19204, 17629, 102],
+                [101, 2117, 7820, 3793, 102, 0, 0, 0, 0],
+            ]
+        ).to(dtype=int)
+        return input_ids
+
+    def test_forward_pretrained(self, utils):
+        input_ids = utils
         encoder = TextEncoder()
-        return encoder, text_input
+        out = encoder(input_ids)
+        expected_sum = -13.6029
+        assert_expected(actual=out.shape, expected=torch.Size([2, 768]), rtol=0, atol=0)
+        assert_expected(
+            actual=out.sum(), expected=torch.as_tensor(expected_sum), rtol=0, atol=1e-4
+        )
 
-    def test_forward(self, start):
-        encoder, text_input = start
-        out = encoder(text_input)
-        expected_sum = -5.8399
-        assert_expected(actual=out.shape, expected=torch.Size([1, 768]), rtol=0, atol=0)
+    def test_forward_untrained(self, utils):
+        input_ids = utils
+        encoder = TextEncoder(pretrained=False, model_config={})
```

Comment:
Should we just set `model_config` as `None` unless you meant to override certain default configs?